src/Compilers/Core/Desktop/AbstractAnalyzerAssemblyLoader.cs

```diff
@@ -0,0 +1,180 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Reflection.Metadata;
+using System.Reflection.PortableExecutable;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis
+{
+    public abstract class AbstractAnalyzerAssemblyLoader : IAnalyzerAssemblyLoader
```

Comment:
@tmeschter i would take the dependency as it is implicit already to help keep our public surface area low.